.talismanrc

```diff
@@ -0,0 +1,19 @@
+fileignoreconfig:
```

Comment:
Should this be git ignored? I know there have been issues in other repos with conflicts as the checksum will be different for different users.

Reply:
Agh, yes, you are right... not sure how that one slipped in there